sw/device/tests/aon_timer_irq_test.c

```diff
@@ -57,8 +60,7 @@ static_assert(configUSE_PREEMPTION == 0,
               "rv_timer may be initialized already by FreeRtos");
 
 static void tick_init(void) {
-  CHECK_DIF_OK(dif_rv_timer_init(
-      mmio_region_from_addr(TOP_EARLGREY_RV_TIMER_BASE_ADDR), &rv_timer));
+  CHECK_DIF_OK(dif_rv_timer_init_dt(rv_timer_dt, &rv_timer));
```

Comment:
I didn't find the definition of `rv_timer_dt` maybe you mean `kDtRvTimer`?

Reply:
The definition is on line 34